cloudinit/osys/windows/util/iphlpapi.py

```diff
@@ -0,0 +1,208 @@
+# Copyright 2015 Cloudbase Solutions Srl
+# Copyright 2015 Canonical Ltd.
+# License TBD
+
+import ctypes
+
+from ctypes import windll
+from ctypes import wintypes
+
+from cloudinit.osys.windows.util import kernel32
+from cloudinit.osys.windows.util import ws2_32
+
```

Comment:
Same question about comments, these are better, but still unsure about some of them (what is ``GAA_``??)